vagrantfile-windows_2019_core.template

```diff
@@ -0,0 +1,61 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.require_version ">= 1.6.2"
+
+Vagrant.configure("2") do |config|
+    config.vm.define "vagrant-windows-2016"
```

Comment:
Can you please update the strings 2016 -> 2019 in these two lines?

Reply:
fixed and waiting for appveyor.